### ISSUE TITLE
Multiple endpoint support?

### ISSUE THREAD

**Issue by dopaminereceptionist** [rqlite/pyrqlite#26]
It would be nice if I could connect to whichever endpoint is online. Something like this:

```
connection = dbapi2.connect(
    hosts=['db1', 'db2', 'db3'],
    port=4001,
)
```

Otherwise, endpoint has to be shared in some sort of failover by multiple nodes in order to avoid a single point of failure.

**Comment by otoolep** [comment]
Agreed, this is something the CLI does and client libraries should too if possible.
